templates/community-viz/README.md

```diff
@@ -25,38 +25,38 @@ directory.
 
 ## Using this template
 
-To create a new community visualization using this template, run the command
+To create a new community visualization based on this template, run the command
 
 ```bash
 npx @google/dscc-gen --project_choice community-viz
 ```
 
 At the end of the setup flow, you will have a minimal working visualization and
-have set GCS buckets for a dev and prod version.
+have set GCS buckets for a dev and prod version. Edit `src/index.js` to see changes.
 
-### Writing the viz
+### Update your local data
 
-1. Define the dimensions and metrics your visualization requires in
+1. Update the dimensions and metrics your visualization requires in
    `src/index.json`
-2. Run the commands `npm run build:dev and; npm run push:dev` to build and
+1. Run the commands `npm run updateMessage -- --format object` to build and
    deploy your visualization to your "dev" bucket.
-3. [Create a new report][datastudio] and connect to the dataset you want to use
+1. [Create a new report][datastudio] and connect to the dataset you want to use
    for your sample message. I'd recommend using a subset of your data, to keep
    the size of the message small.
-4. Use your "dev bucket" to add this visualization to your report. It will
+1. Use your "dev bucket" to add this visualization to your report. It will
    display div with the `data` returned by the [ds-component] helper library.
-5. Copy the `data` in the visualization and replace the empty object in
-   `src/localData.js`. This is the "local data" that you will develop with.
+1. Copy the `data` in the visualization and replace the empty object in
+   `scripts/data/localData.js`. This is the "local data" that you will develop with.
 
 
 ### Local development workflow
 
 To develop locally:
 
 1. Change `const LOCAL` to `true` in `src/index.js`.
-2. Run `npm run start` to start a local server. A browser tab should open with
+1. Run `npm run start` to start a local server. A browser tab should open with
    the visualization you just deployed in Data Studio.
-3. Make changes in `src/index.js` and `src/index.css`, save the changes, and see
+1. Make changes in `src/index.js` and `src/index.css`, save the changes, and see
```

Comment:
Are we sure css reloading works? I haven't tried it yet, but I know I've had issues with that in the past with webpack.

Reply:
Yes, I tried it.